src/parser/type_checking.c

```diff
@@ -12120,7 +12120,7 @@ pt_upd_domain_info (PARSER_CONTEXT * parser, PT_NODE * arg1, PT_NODE * arg2, PT_
 	{
 	case PT_TYPE_CHAR:
 	  dt->info.data_type.precision = ((dt->info.data_type.precision > DB_MAX_CHAR_PRECISION)
-					  ? DB_MAX_CHAR_PRECISION : dt->info.data_type.precision);
```

Comment:
I'd like to confirm that it is related to db_value_domain_init() sets the precision according to the type and DB_DEFAULT_PRECISION. Am I understanding correctly?
If so, this behavior is tough to connect this concept if you don't know the context about it. Describing this behavior by a comment seems to be required here.

Reply:
Ok, I will comment the context.

Reply:
Please add a comment in the source file.